### PR TITLE
Matrix reasoning button audio

### PR DESCRIPTION
**Pull request by zwatson2001**
Added the neutral 'select' audio to the response buttons and removed the audio prompts after the practice trials.

### CODE REVIEW

File: task-launcher/src/tasks/shared/trials/afcStimulus.js [hunk 1]
```diff
@@ -46,7 +46,7 @@ function getStimulus(trialType) {
   if (!mediaAssets.audio[camelize(stim.audioFile)]) return mediaAssets.audio.nullAudio;
   // all tasks should have the replay button play whatever is in stim.audioFile (might be just prompt/instructions)
 
-  if (stim.task === 'Mental Rotation' && stim.notes !== 'practice' && stim.trialType !== 'instructions') {
+  if ((stim.task === 'Mental Rotation' || stim.task === 'Matrix Reasoning') && stim.notes !== 'practice' && stim.trialType !== 'instructions') {
     return mediaAssets.audio.nullAudio;
```
Comment (asengupta3) [inline task-launcher/src/tasks/shared/trials/afcStimulus.js:50]: I think we should remove the nullAudio logic at some point. @Zio-4 this was done for some default work right? @zwatson2001 why do you want to return nullAudio in the case of matrix reasoning?

Comment (zwatson2001) [reply task-launcher/src/tasks/shared/trials/afcStimulus.js:50]: That was to remove the audio prompt for matrix reasoning once the instructions and practice trials are over, but I could look into doing that a different way if that would be better.

Comment (kachergis) [reply task-launcher/src/tasks/shared/trials/afcStimulus.js:50]: yes, for tasks such as matrix reasoning and mental rotation it is annoying to hear the same prompt repeated over and over for many trials. I'm fine with the nullAudio solution, but @asengupta3 if you prefer another approach please let us know.

Comment (asengupta3) [reply task-launcher/src/tasks/shared/trials/afcStimulus.js:50]: I think we should get away from `jsPsychAudioMultiResponse` and change it to `jsPsychHTMLMultiResponse` and let the audio play in the on_load function. That will give us more control over overlapping audio etc. But that can be a separate PR. For now, let's merge this.

**Review by Zio-4 — APPROVED** [review]

**Pull request merged**